workspace/__lib__/xod/debug/table-log/patch.xodp

```diff
@@ -0,0 +1,46 @@
+{
+  "description": "Shows incoming values in the realtime, when a debug session is active",
+  "nodes": [
+    {
+      "id": "B1iYZOzoZ",
+      "position": {
+        "units": "slots",
+        "x": 1,
+        "y": 1
+      },
+      "type": "xod/patch-nodes/not-implemented-in-xod"
+    },
+    {
+      "id": "rkWILuNyu",
+      "label": "PUSH",
+      "position": {
+        "units": "slots",
+        "x": 3,
+        "y": 0
+      },
+      "type": "xod/patch-nodes/input-pulse"
+    },
+    {
+      "boundLiterals": {
+        "__out__": "On Boot"
```

Comment:
This should be "Never" by default. Otherwise we always start with the second sheet